### PR TITLE
Typo on generated-code.md

### PR DESCRIPTION
**Pull request by ckiosidis**

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @ckiosidis isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

File: docs/reference/java/generated-code.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ channel)` static method.
 
 #### Unary
 
-The aynschronous stub signature for a unary RPC method `unaryExample`:
+The asynschronous stub signature for a unary RPC method `unaryExample`:
```
Comment (jtattermusch) [inline docs/reference/java/generated-code.md]: there's still a typo in "asynchronous". also below.

**Comment by ckiosidis** [comment]
@jtattermusch great, i'll take care of that later today.

**Comment by hsaliak** [comment]
@ckiosidis this PR is blocked on signing the CLA, would you be able to take care of that?

**Comment by ckiosidis** [comment]
@dapengzhang0  @hsaliak @jtattermusch 
it looks better now

**Pull request merged**